cmd/ocm/list/rhRegion/cmd.go

```diff
@@ -0,0 +1,53 @@
+package rhRegion
+
+import (
+	"fmt"
+
+	"github.com/openshift-online/ocm-cli/pkg/config"
+	"github.com/openshift-online/ocm-cli/pkg/ocm"
+	"github.com/openshift-online/ocm-cli/pkg/rhRegion"
+	"github.com/spf13/cobra"
+)
+
+var Cmd = &cobra.Command{
+	Use:     "rh-regions",
+	Short:   "List available OCM regions",
+	Long:    "List available OCM regions",
+	Example: `  # List all supported OCM regions ocm list rh-regions`,
+	RunE:    run,
+}
+
+func run(cmd *cobra.Command, argv []string) error {
+	// Load the configuration file:
+	cfg, err := config.Load()
+	if err != nil {
+		return fmt.Errorf("Can't load config file: %v", err)
+	}
+	if cfg == nil {
+		return fmt.Errorf("Not logged in, run the 'login' command")
+	}
+
+	connection, err := ocm.NewConnection().Build()
+	if err != nil {
+		return err
+	}
+	defer connection.Close()
+
+	// Check that the configuration has credentials or tokens that don't have expired:
+	armed, reason, err := cfg.Armed()
```

Comment:
i dont think we need the armed check or an ocm-sdk ConnectionBuilder since ocm-shards is public (anonymous). so i think we can boil this command down to just:
```

func run(cmd *cobra.Command, argv []string) error {
	// Load the configuration file:
	cfg, err := config.Load()
	if err != nil {
		return fmt.Errorf("Can't load config file: %v", err)
	}
	if cfg == nil {
		return fmt.Errorf("Not logged in, run the 'login' command")
	}

	regions, err := rhRegion.GetRhRegions(cfg.URL)
	if err != nil {
		return fmt.Errorf("Failed to get OCM regions: %v", err)
	}

	for regionName := range regions {
		fmt.Println(regionName)
	}
	return nil
```

Reply:
Ack. Removed the check